charts/spire/charts/spiffe-oidc-discovery-provider/templates/tests/test-keys.yaml

```
@@ -0,0 +1,87 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "spiffe-oidc-discovery-provider.fullname" . }}-test-keys"
+  namespace: {{ include "spiffe-oidc-discovery-provider.namespace" . }}
+  labels:
+    {{- include "spiffe-oidc-discovery-provider.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  {{- with .Values.podSecuritycontext }}
```

The bug breaking the tests seems to be here:
```suggestion
  {{- with .Values.podSecurityContext }}
```